docs/src/design/expressions.md

```diff
@@ -0,0 +1,130 @@
+# Expressions
+
+Expressions are *domain specific language (DSL)* built by Mapbox for
+vector styles. Mapbox Vector Style is used in Mapbox Vector Tiles.
```

Comment:
I'm confused what Mapbox Vector Style has to do with Mapbox Vector Tiles? Styles describe how to render any data (including, but not limited to MVTs). Additionally, there are no styles **in** MVTs.

Perhaps you mean something like the following?

```suggestion
vector styles. Mapbox GL Style is used in MapLibre GL renderers.
```

Reply:
Ah good point. The thing wanted to be explicit about is right now the renderer's definition of Map Style schema is limited to Mapbox Vector Style. There are other style schemas too such as Mapzen.

https://stevebennett.me/2017/08/23/openstreetmap-vector-tiles-mixing-and-matching-engines-schemas-and-styles/

Reply:
And on the the topic of *understanding of reality*, I whole heartedly agree. I wanted to start somewhere. I would rather be wrong than looking at a vast empty. :D